subprojects/docs/src/docs/userguide/nativeBinaries.xml

```diff
@@ -311,7 +311,7 @@
                 of the "operators" component, you would run the <userinput>assembleDependentsOperatorsPassingStaticLibrary</userinput> task:
             </para>
             <sample id="nativeAssembleDependentComponents" dir="native-binaries/cunit" title="Assemble components that depends on the passing/static binary of the operators component">
-                <output args="assembleDependentsOperatorsPassingStaticLibrary"/>
+                <output args="assembleDependentsOperatorsPassingStaticLibrary --max-workers=1"/>
```

Comment:
Yeah, the "output" tag causes the samples testing to run the build and compare the output to what is stored in docs/src/samples/userguideOutput.  It has to match the output exactly to pass the test.  If we run in parallel, sometimes the ordering of independent tasks are different and it doesn't match, so we set max workers to 1 to ensure that this isn't flaky.  This is something we should think about as we make more stuff parallel by default and either make the output testing more robust (i.e. somehow handle the ordering differences) or always run the output tests with max workers set to 1.